lxc/remote.go

```diff
@@ -24,18 +26,20 @@ func (c *remoteCmd) usage() string {
 	return gettext.Gettext(
 		"Manage remote LXD servers.\n" +
 			"\n" +
-			"lxc remote add <name> <url>        Add the remote <name> at <url>.\n" +
-			"lxc remote remove <name>           Remove the remote <name>.\n" +
-			"lxc remote list                    List all remotes.\n" +
-			"lxc remote rename <old> <new>      Rename remote <old> to <new>.\n" +
-			"lxc remote set-url <name> <url>    Update <name>'s url to <url>.\n" +
-			"lxc remote set-default <name>      Set the default remote.\n" +
-			"lxc remote get-default             Print the default remote.\n")
+			"lxc remote add <name> <url> [--password=PASSWORD]  Add the remote <name> at <url>.\n" +
+			"lxc remote remove <name>           	   	    Remove the remote <name>.\n" +
+			"lxc remote list                    	   	    List all remotes.\n" +
+			"lxc remote rename <old> <new>      	   	    Rename remote <old> to <new>.\n" +
+			"lxc remote set-url <name> <url>    	   	    Update <name>'s url to <url>.\n" +
+			"lxc remote set-default <name>      	   	    Set the default remote.\n" +
+			"lxc remote get-default             	   	    Print the default remote.\n")
 }
 
-func (c *remoteCmd) flags() {}
+func (c *remoteCmd) flags() {
+	gnuflag.StringVar(&c.password, "password", "", gettext.Gettext("Remote admin password"))
+}
 
-func addServer(config *lxd.Config, server string, addr string) error {
+func addServer(config *lxd.Config, server string, addr string, opts map[string]interface{}) error {
```

Comment:
On Tue, May 12, 2015 at 07:24:09AM -0700, Ben Booth wrote:
> >  
> > -func addServer(config *lxd.Config, server string, addr string) error {
> > +func addServer(config *lxd.Config, server string, addr string, opts map[string]interface{}) error {
> 
> Having problems with the pot file. Can you offer any assistance?

Yep, you should be able to run 'make i18n' and commit the result (no
merging or other thinking required).

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/643/files#r30138410